compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/Patterns.cpp

```diff
@@ -20,6 +20,29 @@ namespace mlir::iree_compiler::IREE::Flow {
 
 namespace {
 
+/// Converts linalg.fill ops into flow.tensor.splat ops.
```

Comment:
Wait, we should already have this?

Reply:
It was moved from lower in the file

Reply:
Oh NM... looks like you moved this.

Reply:
it was out of order - this just moved it up